superset/datasets/commands/exceptions.py

```diff
@@ -191,3 +191,7 @@ class DatasetAccessDeniedError(ForbiddenError):
 
 class DatasetDuplicateFailedError(CreateFailedError):
     message = _("Dataset could not be duplicated.")
+
+
+class DatasetUnAllowedDataURI(ForbiddenError):
```

Comment:
nit: how do you feel about `DatasetDisallowedDataURI` or `DatasetForbiddenDataURI`?